docs/cluster-large.md

```diff
@@ -0,0 +1,56 @@
+# Kubernetes Large Cluster
+
+## Support
+At v1.0, Kubernetes supports clusters up to 100 nodes with 30-50 pods per node and 1-2 container per pod (as defined in the [1.0 roadmap](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/roadmap.md#reliability-and-performance)).
+
+## Setup
+
+Normally the number of nodes in a cluster is controlled by the the value `NUM_MINIONS` in the platform-specific `config-default.sh` file (see [GCE's `config-default.sh`](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/config-default.sh), for example).
+
+Simply changing that value to something very large, however, may cause the setup script to fail for many cloud providers. A GCE deployment, for example, will run in to quota issues and fail to bring the cluster up.
+
+When setting up a large Kubernetes cluster, the following must be taken into consideration.
+
+### Quota Issues
+
+To avoid running into cloud provider quota issues, when creating a cluster with many nodes, consider:
+* Increase the quota for things like CPU, IPs, etc.
+  * In GCE, for example, you'll want to increase the quota for:
+    * CPUs
+    * VM instances
+    * Total persistent disk reserved
+    * In-use IP addresses
+    * Firewall Rules
+    * Forwarding rules
+    * Routes
+    * Target pools
+* Gating the setup script so that it brings up new node VMs in smaller batches with waits in between, because some cloud providers limit the number of VMs you can create during a given period.
```

Comment:
Perhaps point out where the Quota link is in the UI? Does it have a direct link?